deployment/docker/REQUIREMENTS.txt

```diff
@@ -34,4 +34,4 @@ reportlab==3.4.0
 xhtml2pdf==0.2b1
 django-modeltranslation
 django-colorfield
-WeasyPrint
+WeasyPrint==0.42.3
```

Comment:
hi @rduivenvoorde the current code in develop is already fixed this. Could you pull the latest code please? Sorry I missed your PR..